grin/grin.cabal

```diff
@@ -244,7 +244,7 @@ library
     optparse-applicative,
     directory,
     pretty-simple,
-    functor-infix,
```

Comment:
I looked into getting `functor-infix` to increase its bounds, but the author is unresponsive, so I think it's best to switch libraries.

I prefer `composition-extra`'s approach anyway. No TH necessary.